lib/tasks/account.rake

```diff
@@ -16,5 +23,33 @@ namespace :hyku do
         Rails.logger.error "The operation could not be completed"
       end
     end
+
+    desc 'destroy an account and all the data within'
+    task :cleanup, [:name] => [:environment] do |_t, args|
+      account = Account.find_by(name: args[:name])
+      if account.present? && check_confirmation
+        if CleanupAccountJob.perform_now(account)
+          puts "Account successfully deleted"
+          exit 0
+        end
+      end
+      puts "Could not cleanup the tenant"
+      exit 1
+    end
+  end
+end
+
+def check_confirmation
+  unless ENV['CONFIRM'] == 'yes'
+    $stderr.puts <<-EOC
+WARNING: This process will destroy all data for this tenant in:
+DB: All tables
+Fedora: #{ActiveFedora.fedora.build_connection.http.url_prefix}
+Solr: #{ActiveFedora.solr_config[:url]}
+Redis: #{Sidekiq.redis { |c| c._client.options.values_at(:host, :port, :db).join(':') }}
```

Comment:
Are these accurate for the tenant or do you need to `AccountElevator.switch!` before doing this?

Reply:
good point, it only shows the generic connection info right now. I will make another PR for it.